js/views/modals/Settings/Advanced/Settings.js

```diff
@@ -0,0 +1,108 @@
+import app from '../../../../app';
+import loadTemplate from '../../../../utils/loadTemplate';
+import baseVw from '../../../baseVw';
+import Appearance from './Appearance';
+import Transactions from './Transactions';
+import Server from './Server';
+import SMTPIntegration from './SMTPIntegration';
+import AppearanceSettings from '../../../../models/AppearanceSettings';
+import TransactionSettings from '../../../../models/TransactionSettings';
+import ServerSettings from '../../../../models/ServerSettings';
+import SMTPIntegrationSettings from '../../../../models/SMTPIntegrationSettings';
+
+export default class Advanced extends baseVw {
+  constructor(options = {}) {
+    super({
+      className: 'settingsAdvanced',
+      ...options
+    });
+
+    this.options = options;
+    this.settings = app.settings.clone( );
+
+
+    self.app = app;
+
+    this.listenTo(this.settings, 'sync', () => app.settings.set(this.settings.toModelFormatJSON()));
+    this.listenTo(this.settings, 'save', () => app.settings.set(this.settings.toModelFormatJSON()));
```

Comment:
There is no 'save' event on models.

We have made a decision to only update global models (like app.settings) upon 'sync'. The idea is that we only want them updated once the data has successfully been updated on the server. We wouldn't want to update them before that, because if the call fails on the server, we'd have to revert, which could be a bit hairy since, in theory, there can be more than one pending Save request.